src/pages/AddServers.js

```diff
@@ -150,11 +151,20 @@ class AddServers extends BaseUpdateWizardPage {
   // values, then set size is the same as the array list length,
   // otherwise the list contains duplicate values.
   hasDuplicates = (arrayList) => {
-    return (new Set(arrayList)).size !== arrayList.length;
+    if(isEmpty(arrayList)) {
+      return false;
+    }
+    // filter out empty items
+    let cleanList = arrayList.filter(item => !isEmpty(item));
+    if(isEmpty(cleanList)) {
+      return false;
+    }
+
+    return (new Set(cleanList)).size !== cleanList.length;
```

Comment:
A comment here would be helpful. It is not immediately obvious that the point of creating a Set is to remove duplicates.

Reply:
comments are at  the top of the method..I can move it down

Reply:
Thanks, that is better.